x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.test.tsx

```diff
@@ -178,11 +179,11 @@ describe('use cases add to existing case modal hook', () => {
     });
 
     AllCasesSelectorModalMock.mockImplementation(({ onRowClick }) => {
```

Comment:
I was a bit confused by this. This line was creating the issue and the test passes regardless. Isn't the `onRowClick` the case selection action? And if the test is `when a case is not selected` should this function be called? It seems to require `{ id: 'test' } as Case` as parameters.